test/unit/median_oracle.js

```diff
@@ -77,7 +77,8 @@ contract('MedianOracle:pushReport', async function (accounts) {
     expect(await chain.isEthException(oracle.pushReport(1000000000000000000, { from: A }))).to.be.true;
     oracle.addProvider(A, { from: deployer });
     r = await oracle.pushReport(1000000000000000000, { from: A });
```

Comment:
can we create a separate test for this?

I think the original test is `should only push from authorized source`